### PR TITLE
React Native SDK

### PR DESCRIPTION
**Pull request by laves**

### CODE REVIEW

File: demo/react-native/App.tsx [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    });
+  }
+
+  _stopProcessing() {
+    this.setState({
+      buttonDisabled: true,
+    });
+
+    this._picovoiceManager?.stop().then((didStop)=>{
+      if(didStop){
```
Comment (dbartle) [inline demo/react-native/App.tsx:144]: run code formatter

File: demo/react-native/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "picovoice-react-native-demo",
+  "version": "1.0.0",
```
Comment (dbartle) [inline demo/react-native/package.json:3]: bump to 1.1.0?

File: sdk/react-native/README.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+## Compatibility
+
+This binding is for running Picovoice on **React Native 0.62.2+** on the following platforms:
+
+- Android 4.1+ (API 16+)
+- iOS 9.0+
+
+## Installation
+
+To start install be sure you have installed yarn and cocoapods. Then add these native modules to your React Native project.
```
Comment (dbartle) [inline sdk/react-native/README.md:24]: "To start the installation"

File: sdk/react-native/README.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+```javascript
+let didStop = await this._picovoiceManager.stop();
+```
+
+Once the app is done with using PicovoiceManager, be sure you explicitly release the resources allocated for it:
+```javascript
+this._picovoiceManager.delete();
+```
+
+As you may have noticed, there is no need to deal with audio capture to enable intent inference with PicovoiceManager.
```
Comment (dbartle) [inline sdk/react-native/README.md:171]: Delete "As you may have noticed"

File: sdk/react-native/README.md [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+let buffer = getAudioFrame();
+
+try {
+    await this._picovoice.process(buffer);
+} catch (e) {
+    // handle error
+}
+```
+
+For process to work correctly, the audio data must be in the audio format required by Picovoice.
```
Comment (dbartle) [inline sdk/react-native/README.md:229]: Add backticks around 'process'

**Review by dbartle — APPROVED** [review]

File: sdk/react-native/src/picovoice.tsx [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+        this._isWakeWordDetected = true;
+        this._wakeWordCallback(keywordIndex);
+      }
+    } else {
+      const result = await this._rhino.process(frame);
+      if(result['isFinalized'] === true){
+        this._isWakeWordDetected = false;
+
+        // format result in native module did not maintain order
+        let formattedInference;
```
Comment (kenarsa) [inline sdk/react-native/src/picovoice.tsx:101]: is there a guarantee that this keeps the order? If we need order then we should put them in an array or just handle it in the demo manually

Comment (laves) [reply sdk/react-native/src/picovoice.tsx:101]: This block basically just makes it more printer-friendly when using JSON stringify and removes isFinalized, which is used internally in PicovoiceManager. If we think this is unnecessary for the SDK, I can definitely move the formatting to the demo, but I still may want to remove isFinalized here to reduce confusion.

**Pull request merged**